### PR TITLE
Adop 415 error handling

### PR DESCRIPTION
**Pull request by dilipsamrahmcts**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

https://tools.hmcts.net/jira/browse/ADOP-415

**Does this PR introduce a breaking change?** (check one with "x")

- [ ] Yes
- [ ] No

### CODE REVIEW

File: src/main/steps/error/error.njk [hunk 1]
```diff
@@ -2,12 +2,23 @@
...
+        </p>
+        <p class="govuk-body">
+        {{line4}}
+        </p>
+    {% endif %}
+    {% if StatusCodes === StatusCodes.INTERNAL_SERVER_ERROR %}
+    <p class="govuk-body">
+        {{info}}
+        </p>
+    {% endif %}
```
Comment (abidfs) [inline src/main/steps/error/error.njk:23]: alignment is incorrect

File: src/main/steps/error/error.njk [hunk 2]
```diff
@@ -2,12 +2,23 @@
...
   <h1 class="govuk-heading-xl">{{ title }}</h1>
-  <p class="govuk-body">{{ info }}</p>
-  <p class="govuk-body">{{ youCan }}</p>
-  <ul class="govuk-list govuk-list--bullet">
-    <li>{{ goBack }}</li>
-    <li>
-      <a class="govuk-link" href="/logout">{{ signInAgain }}</a>
-    </li>
-  </ul>
+    {% if StatusCodes === StatusCodes.NOT_FOUND %}
```
Comment (abidfs) [inline src/main/steps/error/error.njk:5]:
```suggestion
    {% if statusCode === StatusCodes.NOT_FOUND %}
```

File: src/main/steps/error/error.njk [hunk 3]
```diff
@@ -2,12 +2,23 @@
...
+        {{line2}}
+        </p>
+        <p class="govuk-body">
+        {{line3}}
+        </p>
+        <p class="govuk-body">
+        {{line4}}
+        </p>
+    {% endif %}
+    {% if StatusCodes === StatusCodes.INTERNAL_SERVER_ERROR %}
```
Comment (abidfs) [inline src/main/steps/error/error.njk:19]:
```suggestion
    {% if statusCode === StatusCodes.INTERNAL_SERVER_ERROR %}
```

**Review by abidfs — CHANGES_REQUESTED** [review]
minor changes needed

**Review by pankajhmcts — APPROVED** [review]

**Review by abidfs — APPROVED** [review]

**Pull request merged**